docs/releases.md

```diff
@@ -1,5 +1,97 @@
 # DietPi Releases
 
+## December 2021 (Beta Version 7.9) {: #december-2021-version-79 }
+
+### Overview
+
+**DietPi Beta v7.9 is ready for testing ! While we are in beta, we are working at full speed to roll out DietPi improvments and new features.**
```

Comment:
Let's stick with no spaces before punctuation (`!`/`?`). Grammatically it is simply wrong, and I like to be consistent with this 🙂.

Reply:
yes, fixed both